### ISSUE TITLE
USB port possibly prone to breaking off on 800

### ISSUE THREAD

**Issue by sprive** [FujiNetWIFI/fujinet-hardware#9]
Hey all

Just end user board feedback (v1.0 rev)

First -- thanks! :-) 

Second - on a 400/800 the FN USB port is always needed. I suspect this adds more stress (always in) than plugging and unplugging.

My suggestion probably adds too much cost, but at least you can maybe come up with something alternative/cheaper
* Maybe a panel-mount connector
* or through-hole with screw-in wings/tabs like the A121025TR.
Basically... something gives the connector someplace to offload stress. 

I'll attempt to solder a panel mount lead onto the 0.8mm pads (fun!), unless I get lucky and find larger test pads somewhere (I don't see any, but mentioning this thought out loud... maybe there is?).

cheers

**Comment by mozzwald** [comment]
> on a 400/800 the FN USB port is always needed. I suspect this adds more stress (always in) than plugging and unplugging.

Usually, leaving a cable plugged in all the time does no damage. The unplug/plug cycles are what wear it out. Bumping the cable while it was inserted could damage it. There has been only 1 report of MicroUSB port breaking that I am aware of and it was due to poor soldering by the vendor. This is not something that will be changed.

If you are looking for a place to tap into for adding your own 5V connector, D4 or C6 on v1.0 hardware is a good spot with large enough pads.

![Screenshot from 2021-10-18 07-51-16](https://user-images.githubusercontent.com/1191098/137734583-4688fee2-b700-46a2-b121-22bef0cda5de.png)

**Comment by sprive** [comment]
Thanks - you've given me a better place to attempt a panel-mount cable.

As you say _**"Bumping the cable while it was inserted could damage it. This is not something that will be changed."**_

I _hear_ that.... still, please consider the problem further (if another board revision presents itself)?

Even simple holes to either side would allow one to reinforce with a zip tie, or an empty place on the board for a sturdier DC barrel jack.  Thanks again.

**Comment by rcamp5656** [comment]
does not the case have a slot for putting a cable into the Fujinet, or at least mine does, the case takes all of the stress off of the connector if done right

**Comment by sprive** [comment]
> does not the case have a slot for putting a cable into the Fujinet, or at least mine does, the case takes all of the stress off of the connector if done right

Ah, a different perspective. TY. "yes and no" to your question.

Basically the case opening is an oversized hole the USB cable plug goes into. It is not at all a "close" fit. Case design is one used by the vendor, VCC.

(Perhaps another case design would allow a tight fit around the USB part shroud, reinforcing the hole on the inside rather than the same wall thickness everywhere? The USB shroud does protrude 1mm, but a case wall needs to be thicker than that)

I just received DC barrel jacks, threaded kind with a nut. Then for me the USB is only used for flashing updates.

Alternatively, a FN on an 800 doesn't need external power if one buys the Incognito board. Incognito adds a tiny boot delay which means the FN is initialized by the time the OS checks SIO for boot drive.
